eng/build-test.yml

```diff
@@ -0,0 +1,186 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  container: ''
+  testGroup: ''
+  readyToRun: false
+  helixQueues: ''
+  crossrootfsDir: ''
+  # If true, run the corefx tests instead of the coreclr ones
+  corefxTests: false
+  displayNameArgs: ''
+  runInUnloadableContext: false
+  publishTestArtifacts: true
+
```

Comment:
Do you mind adding a default value for `tarCompression`, `archiveType` and `archiveExtension`. IMO we should favor zip.

Reply:
Hmm, I apologize, I'm not getting this, can you please elaborate? At this moment, these are not 'primary' properties, these are 'synthesized variables' (calculated in xplat-job.yml) based on the active OS (zip on Windows, tar.gz anywhere else). Why should we add them to explicit parameters?

Reply:
Correct. My suggestion is to create a default variable, so that if it is not passed it will default to zip.

Reply:
After more thought though, it would fail if not provided, which is probably what we are looking for. I think I am good without the variables added.

Will leave this unresolved in case someone else wants to give their own opinion.